### ISSUE TITLE
🐛 Bug Report: using images with anthropic throws an exception

### ISSUE THREAD

**Issue by coffecup25** [traceloop/openllmetry#1944]
### Which component is this bug for?

Anthropic Instrumentation

### 📜 Description

When sending including an image to anthropics message model the function throws an exception: TextInputSequence must be str.

### 👟 Reproduction steps

Include a base64 encoded image in the messages 

```json
"messages": [
  {
    "role": "user",
    "content": [
      {
        "type": "text",
        "text": "PLACEHOLDER"
      },
      {
        "type": "image",
        "source": {
          "type": "base64",
          "media_type": "image/jpeg",
          "data": "PLACEHOLDER"
        }
      }
    ]
  }
]
 ```

### 👍 Expected behavior

Include usage for images as well without throwing an error. And if there is an exception it should log to stdout rather than raise the exception

### 👎 Actual Behavior with Screenshots

Throws exception: TextInputSequence must be str
Happens when calling anthropic.count_tokens()

I don't understand why that function is used when the usage is included in the response. I monkey-patched it locally to use the usage attribute which solved the problem.

Furthermore even, though the function is wrapped with @dont_throw I got an exception.

### 👀 Have you spent some time to check if this bug has been raised before?

- [X] I checked and didn't find similar issue

### Are you willing to submit PR?

None

**Comment by galkleinman** [comment]
Hey @coffecup25, thanks for reporting this one!

I'm working atm on adding support for images in messages (in general), so i'll have a look at this one too :)

**Comment by coffecup25** [comment]
Great @galkleinman!  Do you any ETA when the library will have the updates?

**Comment by galkleinman** [comment]
@coffecup25 hopefully early next week :)

**Comment by coffecup25** [comment]
Looking forward to the update! Is there a rational for using the count_tokens function or will you also switch over to using the  usage attribute?

**Comment by galkleinman** [comment]
Should be solved in `0.31.1`

**Comment by galkleinman** [comment]
Closing. Should be solved in 0.31.1.
